x-pack/packages/kbn-elastic-assistant/impl/assistant/conversations/conversation_selector/index.tsx

```diff
@@ -33,9 +33,11 @@ interface Props {
   defaultConnectorId?: string;
   defaultProvider?: OpenAiProviderType;
   selectedConversationId: string | undefined;
-  onConversationSelected: (conversationId: string) => void;
+  onConversationSelected: (conversationId: string, title?: string) => void;
```

Comment:
Consider named parameters (in general), e.g.

```ts
  onConversationSelected: ({
    conversationId,
    title,
  }: {
    conversationId: string;
    title?: string;
  }) => void;
```

over

```ts
onConversationSelected: (conversationId: string, title?: string) => void;
```

because the latter style prevents the compiler from catching bugs like the following:

```ts
onConversationSelected('title', 'id'); // no compiler error
```